library/src/main/java/com/datatorrent/lib/io/jms/JMSBase.java

```diff
@@ -143,7 +172,7 @@ public void setConnectionFactoryProperties(Map<String, String> connectionFactory
   @Deprecated
   public void setUser(String user)
```

Comment:
Another option is to provide the connectionFactoryClass helper for now and take up deprecation discussions after these changes are included.